### PR TITLE
Adding mac CI

### PR DESCRIPTION
**Pull request by fao89**
[noissue]

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -13,10 +13,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        os: [ubuntu-latest, macos-latest]
```
Comment (fao89) [inline .github/workflows/ci.yaml:16]: https://github.com/actions/runner-images/blob/macOS-11/20220906.1/images/macos/macos-11-Readme.md

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -38,21 +39,33 @@ jobs:
...
+        if: matrix.env.TEST == 'podman' && matrix.os == 'macos-latest'
+        run: |
+          brew install podman
+          sudo pip install httpie podman-compose
+          podman --version
+
+      - name: (Mac) Install docker compose
+        if: matrix.env.TEST == 'docker' && matrix.os == 'macos-latest'
+        run: |
+          brew install docker-compose
```
Comment (fao89) [inline .github/workflows/ci.yaml:67]: I have no idea on how to make it work on mac

Comment (mikedep333) [reply .github/workflows/ci.yaml:67]: When I get back home, I can setup remote access to my Macbook.

You could also try using a tmate debug shell:
https://github.com/marketplace/actions/debugging-with-tmate

**Comment by newswangerd** [comment]
I don't think that podman on mac is currently possible. There are a lot of caveats around volumes and networks for podman-machine atm.

**Comment by bmbouter** [comment]
@newswangerd @dkliban are either of you able to review this? If it passes I'm hoping we can get it merged soon.

**Review by newswangerd — APPROVED** [review]

**Pull request merged**